src/document/DocumentManager.ts

```diff
@@ -245,6 +245,9 @@ export class DocumentManager {
       this.model.uiState.setSelectedSidebarItem(undefined);
       this.model.uiState.setSelectedNavbarItem(-1);
     });
+    hotkeys("ctrl+o,command+o", () => {
+      this.saveFileDialog();
```

Comment:
This opens the save file dialog, not the open file dialog.

Reply:
```suggestion
      invoke("open_file_dialog");
```

Reply:
yeah not sure if it is possible with the current setup. ill need to refactor things around from mobx context to documentManager.